app/controllers/authenticate_controller.rb

```diff
@@ -122,6 +123,21 @@ def authenticator_input
     )
   end
 
+  def tiny_authenticator_input
```

Comment:
so id this the same as `authenticator_input` but without credentials? can we call it `authenticator_input_without_credentials` to be more explicit?

Reply:
The change is subject for next PR. I'll take your remark forward :-)